src/services/refactors/extractSymbol.ts

```diff
@@ -740,6 +740,8 @@ namespace ts.refactor.extractSymbol {
         }
 
         const { body, returnValueProperty } = transformFunctionBody(node, exposedVariableDeclarations, writes, substitutions, !!(range.facts & RangeFacts.HasReturn));
+        suppressLeadingAndTrailingTrivia(body);

```

Comment:
So if you create a fresh node that re-use tokens, do you suppressLeadingAndTrailingTrivia on the new node or the old node?

Reply:
As we discussed offline, you probably want to suppress it on the new node.  Note that the token shouldn't appear in two places in the tree - that will mess up the positions.  You have to make a copy.